### PR TITLE
Update failing tests

### PR DESCRIPTION
**Pull request by csteegz**
**Description**:
Update failing functional tests for ORT runtime

**Motivation and Context**
ORTServer wasn't failing on failed integration tests, which allowed some broken integration tests to be introduced. Fix those tests.

### CODE REVIEW

File: onnxruntime/test/server/unit_tests/http_routes_tests.cc [hunk 1]
```diff
@@ -36,6 +36,10 @@ TEST(HttpRouteTests, PostRouteTest) {
       std::make_tuple(http::verb::post, "/v1/models/models/versions/45:predict", "models", "45", "predict", http::status::ok),
       std::make_tuple(http::verb::post, "/v1/models/??$$%%@@$^^/versions/45:predict", "??$$%%@@$^^", "45", "predict", http::status::ok),
       std::make_tuple(http::verb::post, "/v1/models/versions/versions/45:predict", "versions", "45", "predict", http::status::ok)};
+      std::make_tuple(http::verb::post, "/score", "", "", "", http::status::ok)};
```
Comment (NonStatic2014) [inline onnxruntime/test/server/unit_tests/http_routes_tests.cc]: `}` is unexpected :-)

**Comment by vinitra** [comment]
\azp run

**Pull request merged**